docs/module/routing.md

```diff
@@ -433,11 +433,12 @@ A static route entry's **nexthop** attribute specifies the next hop used to reac
 
 * **ipv4** -- an IPv4 next hop (an address, not a prefix)
 * **ipv6** -- an IPv6 next hop (an address, not a prefix)
+* **gateway** (boolean) -- the next hop is the default gateway. This entry will be resolved into a list containing all directly connected default gateways in the specified next-hop VRF (a node might use more than one default gateway to reach a destination).
 * **node** -- The next hop is the specified node. The node name will be resolved into a list of directly connected next-hops or the control-plane endpoint of a distant node (allowing you to specify recursive static routes).
```

Comment:
For a node with ```ipv6: True``` on its interface, would it make sense to have ```node``` resolve to the interface towards that node?

Reply:
Not at the moment. That would require IPv6 proxy-arp on the other node. Also, static routes pointing to interfaces (with no next hops) are a REALLY BAD idea. Here are the impacts https://blog.ipspace.net/2009/10/follow-up-interface-default-route/ and this is how I found it out https://blog.ipspace.net/2009/10/my-stupid-moments-interface-default/